lib/Pseudo_QC_String.qpp

```diff
@@ -296,6 +296,38 @@ bool <string>::intp() [flags=CONSTANT] {
    return (isdigit(c)) ? true : false;
 }
 
+//! Converts the string to an integer value with respect to the <b>base</b> (does the same as Qore::int(string str, softint base)). If a character is out of range of the corresponding <b>base</b> only the substring preceeding the character is taken into account. Base 0 means to interpret as a code literal, so that the actual base is 8, 10, or 16.
```

Comment:
Move the detailed explanation about "base" argument etc. (after the first sentence) to the second (detail) part of the documentation. Do the same with the int() function.